data/card_catalogue/unique-1.json

```diff
@@ -214,7 +264,10 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Possess a non-magic object within 5 tiles of your body until you lose concentration.",
+      "rules": {
+        "active": "Possess a non-magic object within 5 tiles of your body until you lose concentration.",
```

Comment:
I think this is the sort of situation where we can say 'magic' as nobody would be upset with us saying the 'Pissed-off-and-well-oiled Murderbot' or 'Plot-Significant Mainframe Computer' had some magic in it even though it's an object (also for those examples as Cardmaster is my time away from Cloud Computing and I could do without it being mentioned then).